File: Naloga 28/naloga_28.cpp
#include <iostream>

void naloga_28 () {
    unsigned int stevec = 1;
    unsigned int vsota = 0;
    unsigned int stevec_stevilo[2] = {0,0}; // stevilo, stevec
    unsigned int stevilo;
    while (true) {
        std::cin >> stevilo;
        if (stevilo == 0) {break;}
        else if (stevilo > stevec_stevilo[0]) {
            stevec_stevilo[0] = stevilo;
            stevec_stevilo[1] = stevec;
        };
        ++stevec;
        vsota += stevilo;
    };
    std::cout << "najvecje stevilo je: " << stevec_stevilo[0] << ", ki je bilo " << stevec_stevilo[1] << " po vrsti" << std::endl;
    std::cout << "vsota stevil je: " << vsota << std::endl;
};

int main () {
    naloga_28();
    return 0;
};

File: Naloga 28/naloga_28.py
def naloga_28():
    stevec = 1
    vsota = 0
    stevec_stevilo = [0,0] # stevec, stevilo
    while True:
        stevilo = int(input("vnesi stevilo: "))
        if stevilo == 0:
            break
        elif int(stevilo) > stevec_stevilo[1]:
            stevec_stevilo = [stevec, stevilo]
        vsota += stevilo
        stevec += 1
    print(f"najvecje stevilo je: {stevec_stevilo[1]}, ki je bilo {stevec_stevilo[0]} po vrsti")
    print(f"vsota je: {vsota}")
    return
if __name__ == '__main__':
    naloga_28()
